superset-frontend/src/views/App.tsx

```diff
@@ -38,7 +38,10 @@ import { RootContextProviders } from './RootContextProviders';
 setupApp();
 
 const user = { ...bootstrapData.user };
-const menu = { ...bootstrapData.common.menu_data };
+const menu = {
+  ...bootstrapData.common.menu_data,
+  allowedExtensions: bootstrapData.common.allowed_extensions,
```

Comment:
lets just pass the entire `common.conf` object here, I'm using it to pass the if google sheet install key